### ISSUE TITLE
Guest voucher does not work if add_me_to_recipients selected

### ISSUE THREAD

**Issue by stlarose** [filesender/filesender#1802]
Hi,

A JavaScript error arise (and the files are not sent) when a guest click on the Send button if the guest voucher was created with the option "Include me as a recipient". The error from the JS console is:

```
Uncaught ReferenceError: auth is not defined
    at getGuestOption (upload_page.js:83:5)
    at HTMLAnchorElement.<anonymous> (upload_page.js:1874:25)
    at HTMLAnchorElement.dispatch (jquery.min.js:2:40035)
    at v.handle (jquery.min.js:2:38006)
```

Looking at www/js/upload_page.js, we can see that if the option "add_me_to_recipients" is selected, the "getGuestOption" function is called:
```
1872        // confirm that the upload is only intended to the voucher issuer.
                if( getOption( 'add_me_to_recipients' )
                    && !getGuestOption( 'can_only_send_to_me' )
                    && !filesender.ui.transfer.recipients.length )
```

And the "getGuestOption" function use the auth variable which is not defined in this context.

```
function getGuestOption( n )
{
    if(auth == 'guest') {
        return filesender.ui.guest_options[n];
    }
    return false;
}
```
Thanks!

**Comment by monkeyiq** [comment]
Thank you for the detailed report. I will have a fix merged soon. The patch from the above PR should also work in the meantime.

**Comment by monkeyiq** [comment]
This update is released in 2.46.
